lib/config/fileconf.go

```diff
@@ -183,7 +183,7 @@ func MakeSampleFileConfig(flags SampleFlags) (fc *FileConfig, err error) {
 	s.Commands = []CommandLabel{
 		{
 			Name:    "hostname",
-			Command: []string{"hostname"},
+			Command: []string{"/bin/hostname"},
```

Comment:
@webvictim Actually, see Marco's comment above. Seems like this still won't work in all scenarios? I don't think it's such a huge issue, maybe just keep it as is for now?

Reply:
Yeah, if it's not actually solving the issue then this isn't really worth doing.